bash/create-service-principal-2.0.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
```

Comment:
Personally I think we should maintain the 2.0 version since the xplat-cli is already deprecated. Using docker to create sp can avoid version detection job since docker already pack the dependencies. Does it make sense?